packages/react-components/src/AccessibilityScenarios/ListOfScenarios.stories.mdx

```diff
@@ -1,16 +1,24 @@
 import { Meta } from '@storybook/addon-docs';
-import { FullscreenLink } from './utils';
+import { FullscreenLink, APP_TITLE, APP_TITLE_SEPARATOR } from './utils';
 
-<Meta title="Accessibility Scenarios/List of Scenarios" />
+<Meta title={`List of Scenarios${APP_TITLE_SEPARATOR}${APP_TITLE}`} />
```

Comment:
Let's remove the separator and title from here as well, and then the import.

Reply:
Done.